bridge/client/app/_views/ktb-sequence-view/ktb-sequence-view.component.spec.ts

```diff
@@ -31,142 +77,130 @@ describe('KtbEventsListComponent', () => {
-    dataService.loadProjects();
-    const projectsRequest = httpMock.expectOne('./api/controlPlane/v1/project?disableUpstreamSync=true&pageSize=50');
-    projectsRequest.flush({
-      projects: ProjectsMock,
-      totalCount: ProjectsMock.length,
-    });
-
-    fixture.detectChanges();
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore // Ignore private property
+    component.latestDeployments = SequenceMetadataMock.deployments;
+    component.selectedStage = 'staging';
+    component.currentSequence = SequencesMock[0];
 
-    const loadingIndicator = fixture.nativeElement.querySelector('[uitestid=keptn-loadingSequences]');
-    const emptyStateContainer = fixture.nativeElement.querySelector('[uitestid=keptn-noSequences]');
-    const sequenceList = fixture.nativeElement.querySelector('[uitestid=keptn-sequence-view-roots]');
+    // when
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore // Ignore private property
+    component.updateLatestDeployedImage();
 
     // then
-    expect(loadingIndicator).toBeTruthy();
-    expect(emptyStateContainer).toBeFalsy();
-    expect(sequenceList).toBeFalsy();
+    expect(component.currentLatestDeployedImage).toEqual('carts:0.12.3');
   });
 
-  it('should show empty state if no sequences loaded', async () => {
+  it('should not alter service filters if metadata and sequences match', () => {
     // given
-    dataService.loadProjects();
-    const projectsRequest = httpMock.expectOne('./api/controlPlane/v1/project?disableUpstreamSync=true&pageSize=50');
-    projectsRequest.flush({
-      projects: ProjectsMock,
-      totalCount: ProjectsMock.length,
-    });
-
-    const sequencesRequest = httpMock.expectOne(`./api/controlPlane/v1/sequence/${projectName}?pageSize=25`);
-    sequencesRequest.flush({
-      states: [],
-      totalCount: 0,
-    });
-
-    httpMock.verify();
-    fixture.detectChanges();
+    /* eslint-disable @typescript-eslint/ban-ts-comment */
+    /* @ts-ignore */ // Ignore private property
+    component.project = ProjectsMock[0];
+    // @ts-ignore // Ignore private property
+    component.project.sequences = SequencesMock;
+    /* eslint-enable @typescript-eslint/ban-ts-comment */
+    component._seqFilters = activeFilter;
 
-    const loadingIndicator = fixture.nativeElement.querySelector('[uitestid=keptn-loadingSequences]');
-    const emptyStateContainer = fixture.nativeElement.querySelector('[uitestid=keptn-noSequences]');
-    const sequenceList = fixture.nativeElement.querySelector('[uitestid=keptn-sequence-view-roots]');
+    // when
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore // Ignore private property
+    component.mapServiceFilters(SequenceMetadataMock);
 
-    // then
-    expect(loadingIndicator).toBeFalsy();
-    expect(emptyStateContainer).toBeTruthy();
-    expect(sequenceList).toBeFalsy();
+    expect(component._seqFilters).toEqual(activeFilter);
```

Comment:
Both are referencing the same array.
No matter how the filter will be changed (except a new array is assigned), it will be as the same as `activeFilter`